### PR TITLE
app-editors/jedit: restrict to java 1.8

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: app-editors/jedit/jedit-5.4.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+DEPEND="
+	${CP_DEPEND}
+	virtual/jdk:1.8
+	dev-java/ant-contrib:0
+	dev-java/ant-apache-bsf:0
+	dev-java/bsh:0
+	test? (
+		dev-java/ant-junit:0
+		dev-java/hamcrest-library:1.3
+	)"
```
Comment (fordfrog) [inline app-editors/jedit/jedit-5.4.0-r1.ebuild:32]: please comment on why we need the java version restriction

**Comment by fordfrog** [comment]
@vaukai `dev-java/ant-contrib` commits merged, thanks!

File: app-editors/jedit/jedit-5.4.0-r1.ebuild (hunk 1 above)
Comment (vaukai) [reply app-editors/jedit/jedit-5.4.0-r1.ebuild:33]: Done

File: app-editors/jedit/jedit-5.4.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+HOMEPAGE="http://www.jedit.org"
+SRC_URI="mirror://sourceforge/${PN}/${PN}${PV}source.tar.bz2"
+
+LICENSE="BSD GPL-2"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+SLOT="0"
+
+CP_DEPEND="dev-java/jsr305:0"
+
+# Restricting to java 1.8 as it fails to compile with newer java versions
```
Comment (fordfrog) [inline app-editors/jedit/jedit-5.4.0-r1.ebuild:19]: could you please be more specific?

Comment (vaukai) [reply app-editors/jedit/jedit-5.4.0-r1.ebuild:19]: > could you please be more specific?

Could you recommend, is there another way of solving https://bugs.gentoo.org/828559?

**Comment by fordfrog** [comment]
@vaukai thanks!

**Pull request closed without merging**